packages/code-display/src/CodeDisplay.tsx

```diff
@@ -112,4 +123,22 @@ export const CodeDisplay: FC<PrismEditorProps> = ({
       )}
     </Highlight>
   )
+  if (!onChange) {
+    return highlight(code)
```

Comment:
Also L117 can be modified to use the Span from @looker/components

Reply:
We can't use Span as it has default styles which conflict with Prism's styles applied onto a span